scripts/deploy_oracle.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * PRIVATE_KEY='' ts-node scripts/deploy_oracle.ts "ropsten"
+ */
+import { ethers, providers } from 'ethers'
```

Comment:
can we remove this file from the repo? We don't usually need to commit every script we use, sometimes we just write one-off scripts and don't commit them.

If you want to commit it, rename the function & variable names to reflect the correct names. Also line 2 PRIVATE_KEY='' should be PRIVATE KEY={private_key}.